azure-test.sh

```diff
@@ -5,7 +5,10 @@ set -o errexit
 pip install pytest pytest-azurepipelines
 
 pytest --durations=1 --cov=cloudsync  cloudsync/tests cloudsync/oauth/apiserver.py --timeout=300 &
+pid1=$!
 
 pytest --durations=1 --cov=cloudsync --cov-append --cov-report xml cloudsync/tests/test_provider.py --provider=filesystem &
+pid2=$!
 
-wait
+wait $pid1
```

Comment:
just realized we may want to add some print statements around these waits to make failures easier to diagnose.

(this job failed for this PR, seems like a timeout of the apiserver test, not easy to figure out based on the logs though)

Reply:
What print statements do you suggest?